SConstruct

```diff
@@ -58,6 +59,7 @@ AddOption(
     '--qt',
     dest='qt',
     type='string',
+    default='/usr',
```

Comment:
Can you extract this line in a single commit? The other changes are not necessary: 4 spaces in python is the [PEP8 recommendation for indentation](https://www.python.org/dev/peps/pep-0008/#indentation).